internal/cri/server/podsandbox/sandbox_run_other.go

```diff
@@ -29,7 +29,15 @@ import (
 
 func (c *Controller) sandboxContainerSpec(id string, config *runtime.PodSandboxConfig,
 	imageConfig *imagespec.ImageConfig, nsPath string, runtimePodAnnotations []string) (_ *runtimespec.Spec, retErr error) {
-	return c.runtimeSpec(id, "", annotations.DefaultCRIAnnotations(id, "", "", config, true)...)
+	// SandboxPauseImageName is the name of the contain image used to scope pod shared resources used by the pod's containers,
```

Comment:
nit.. lines 32 - 39 here to determine the sandbox image name.. could be placed in a common func in sandbox_run.go

Reply:
Done.